src/Microsoft.AspNet.Mvc.Core/ActionResults/FileStreamResult.cs

```diff
@@ -39,7 +41,7 @@ public FileStreamResult([NotNull] Stream fileStream)
         /// </summary>
         /// <param name="fileStream">The stream with the file.</param>
         /// <param name="contentType">The Content-Type header of the response.</param>
-        public FileStreamResult([NotNull] Stream fileStream, string contentType)
+        public FileStreamResult([NotNull] Stream fileStream, MediaTypeHeaderValue contentType)
```

Comment:
From our perspective files just contain bytes. It's up the application to decide what the best encoding is to use with that file and inform the user appropriately.